src/api/app/views/staging/staged_projects/index.xml.builder

```diff
@@ -0,0 +1,23 @@
+xml.staging_projects do
+  @staging_projects.each do |staging_project|
+    xml.staging_project(name: staging_project.name) do
+      render(partial: 'staging/staging_projects/staged_requests', locals: { staged_requests: staging_project.staged_requests,
```

Comment:
Can we not just leverage the show template instead of repeating every partial which is in it?